app/javascript/utils/form/final_form/Field.js

```diff
@@ -78,7 +92,37 @@ export const CurrencyField = ({ fieldName, validation, id, label, placeholder, m
     validate={validation}
     component='input'
     format={formUtils.formatPrice}
-    parse={(value, name) => identity(value, name, true)}
+    parse={(value, name) => identity(value, name, FieldType.Currency)}
+    formatOnBlur={isDirty}
+  >
+    {({ input, meta }) => (
+      <div className={classNames((label && 'form-group'), (meta.error && meta.touched && 'error') || '')} >
+        <Label
+          label={label}
+          id={id || `form-${fieldName}`}
+          fieldName={fieldName} />
+        <input {...input}
+          id={id || `form-${fieldName}`}
+          placeholder={placeholder || '$0.00'}
+          className={(meta.error && meta.touched && 'error') || ''}
+          maxLength={maxLength}
+          disabled={disabled} />
+        <FieldError meta={meta} />
+      </div>
+    )}
+  </Field>
+)
+
+/**
+ * Field that allows only whole integer percent values.
+ */
+export const PercentField = ({ fieldName, validation, id, label, placeholder, maxLength, disabled, isDirty = true }) => (
```

Comment:
This is basically just copied and pasted from `CurrencyField`, except for the parse and format props in `Field`.